planning_msgs/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ add_message_files(
 add_service_files(
   FILES
   PlannerService.srv
+  StringService.srv
```

Comment:
I'd strongly recommend against a StringService type. It has not semantic meaning thus cannot easily be introspected or reused when discovered on a system or in a bag file.

Reply:
thanks @tfoote , @rikba I propose we create a message of type "polygon" which can contain a vector of holes and use this msg as a request for the service. It would then be representative of what this service actually should do. What do you think?